src/bitcoind.cpp

```diff
@@ -57,6 +58,8 @@ static bool AppInit(int argc, char* argv[])
 {
     bool fRet = false;
 
+    thread_util::Rename("bitcoind");
```

Comment:
In commit "threads: explicitly name main threads" (cf82eda8aec147a0d24ade9263f8df0708ff02bb)

Since OS thread names are now prefixed with "bitcoin-", I guess these will show up as "bitcoin-bitcoind", "bitcoin-bitcoin-qt-init", and ""bitcoin-bitcoin-qt" in top? This seems ok, but a little redundant. You could consider just calling `Rename()` with "main" and "init" so the OS names would be "bitcoin-main" and "bitcoin-init".

Reply:
Done, thanks. Non-internal thread names now look like 
```sh
 $ pstree 12441

bitcoin-init─┬─{bitcoin-addcon}
             ├─{bitcoin-dnsseed}
             ├─4*[{bitcoin-httpwor}]
             ├─{bitcoin-http}
             ├─{bitcoin-init}
             ├─{bitcoin-loadblk}
             ├─{bitcoin-msghand}
             ├─{bitcoin-net}
             ├─{bitcoin-opencon}
             ├─{bitcoin-schedul}
             ├─7*[{bitcoin-scriptc}]
             └─{bitcoin-torcont}
```